ui/src/image_annotator.tsx

```diff
@@ -0,0 +1,356 @@
+import * as Fluent from '@fluentui/react'
+import { B, Id, S, U } from 'h2o-wave'
+import React from 'react'
+import { stylesheet } from 'typestyle'
+import { AnnotatorTags } from './text_annotator'
+import { clas, cssVar, cssVarValue, px } from './theme'
+import { wave } from './ui'
+
+/** Rectangle (box) annotation. */
+interface ImageAnnotatorRect {
+  /** Start X dimension. */
+  x1: U
+  /** End X dimension. */
+  x2: U
+  /** Start Y dimension. */
+  y1: U
+  /** End Y dimension. */
+  y2: U
+}
+
+/** Defines a particular shape to be used for the annotation. */
+interface ImageAnnotatorShape {
+  rect: ImageAnnotatorRect
+}
+
+/** Image annotator tag. */
+interface ImageAnnotatorTag {
+  /** An identifying name for this tag. */
+  name: Id
+  /** Text to be displayed. */
+  label: S
+  /** HEX or RGB color string used as background for highlighted phrases. */
+  color: S
+}
+
+/** Create an annotator item with initial selected tags or no tag for plaintext. */
+interface ImageAnnotatorItem {
+  /** The annotation shape. */
+  shape: ImageAnnotatorShape
+  /** Tag connected to the highlighted section. */
```

Comment:
```suggestion
  /** The `name` of the image annotator tag to refer to for the `label` and `color` of this item. */
```

(Also change the docstring for the text annotator item if possible).